web/gds-user-ui/src/components/ReviewSubmit/index.tsx

```diff
@@ -43,45 +41,40 @@ const ReviewSubmit: React.FC<ReviewSubmitProps> = ({
   isMainNetSent,
   result,
   isTestNetSubmitting,
-  isMainNetSubmitting
+  isMainNetSubmitting,
+  handleJumpToLastStep,
 }) => {
-  const isTestNetSubmitted: boolean = useSelector(getTestNetSubmittedStatus);
-  const isMainNetSubmitted: boolean = useSelector(getMainNetSubmittedStatus);
+  const { certificateStep } = useFetchCertificateStep({ key: StepEnum.ALL });
   const { isOpen, onOpen, onClose } = useDisclosure();
   const isSent = isTestNetSent || isMainNetSent;
-  const [testnet, setTestnet] = useState(false);
-  const [mainnet, setMainnet] = useState(false);
-  const { jumpToLastStep, jumpToStep } = useCertificateStepper();
-  // const navigate = useNavigate();
+  const { jumpToStep } = useCertificateStepper();
+  const { status } = useSubmissionStatus();
   const dispatch = useAppDispatch();
+  
+  // Display a warning box if the user has reached the submit step but not completed the
+  // TRISA implementation step for one of the networks.
+  const mainnetCommonName = certificateStep?.form?.mainnet?.common_name;
+  const mainnetEndpoint = certificateStep?.form?.mainnet?.endpoint;
+  const testnetCommonName = certificateStep?.form?.testnet?.common_name;
+  const testnetEndpoint = certificateStep?.form?.testnet?.endpoint;
+
+  const isMainnetNetworkIncomplete = !mainnetCommonName || !mainnetEndpoint;
+  const isTestnetNetworkIncomplete = !testnetCommonName || !testnetEndpoint;
+
+  const isTestNetSubmitted = status?.data?.testnet_submitted;
```

Comment:
I attempted to resolve the issue by fixing the redux store's state for the submission status. When I completed a registration form, the confirmation modal didn't display the data a user would need. There's a bug I couldn't quite figure out in how the state for the submission status is set, so I decided to retrieve the status from the BE here. I'll looking into the issue in how the state is updated again in another story.